### DISCUSSION TITLE
Publishing to firebase app distribution failed when i used aab, error project not linked

### DISCUSSION THREAD

**Discussion by ekokurniadi** [codemagic-ci-cd/codemagic-docs#2164] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I have a already distribute app with apk, i want to change to appbundle, but i got error when i want to publish my appbundle to firebase app distribution

with error 

Error: This project is not linked to a Google Play account.

Failed to publish /Users/builder/clone/build/app/outputs/bundle/productionRelease/app-production-release.aab to Firebase App Distribution: 
Error: This project is not linked to a Google Play account.
.

Build failed :|

Publishing failed :|
Failed to publish /Users/builder/clone/build/app/outputs/bundle/productionRelease/app-production-release.aab to Firebase App Distribution: 
Error: This project is not linked to a Google Play account

### Expected results

App bundle can published to my firebase app distribution

### Actual results

App bundle can't published to my firebase app distribution with error  Error: This project is not linked to a Google Play account

**Comment by himesh-cm** [comment]
Hi @ekokurniadi,
 
Please note that publishing `aab` to Firebase App Distribution requires it to be linked to Google Play. 
If the project is not linked to a Play Store account, then you can consider publishing apk instead.

Please refer to the link [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before_you_begin) for more information.

**Comment by himesh-cm** [comment]
Hi @ekokurniadi,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
